src/main/scala/it/pagopa/pdnd/interop/uservice/partymanagement/api/impl/PartyApiServiceImpl.scala

```diff
@@ -52,7 +53,7 @@ class PartyApiServiceImpl(
     logger.info(s"Verify organization $id")
 
     val result: Future[Option[Party]] = for {
-      uuid <- id.asUUID.toFuture
+      uuid <- id.toUUID.toFuture
```

Comment:
```suggestion
      uuid <- id.toFutureUUID
```

there is another `StringOps` converter.